src/main/java/com/example/controller/MainMenuController.java

```diff
@@ -29,6 +33,8 @@ public class MainMenuController implements ChangeListener<ApplicationState> {
     private void bindToolbars(){
         graphToolsController.setDrawMenuController(drawMenuController);
         graphToolsController.setSimulationMenuController(simulationMenuController);
+        graphToolsController.setSimulationController(simulationController);
+        graphToolsController.bindButtons();
```

Comment:
bindButtons could be invoked inside setSimulationController, to be sure that they will always be binded to proper controller